### ISSUE TITLE
[Functions]@CosmosDBOutput annotation generating function.json as input binding

### ISSUE THREAD

**Issue by pragnagopa** [Microsoft/azure-maven-plugins#483]
Following function
```
public void CosmosTriggerAndOutput(
            @CosmosDBTrigger(name = "itemIn", databaseName = "%CosmosDBDatabaseName%", collectionName = "ItemCollectionIn", leaseCollectionName = "leases", connectionStringSetting = "AzureWebJobsCosmosDBConnectionString", createLeaseCollectionIfNotExists = true) Object inputItem,
            @CosmosDBOutput(name = "itemOut", databaseName = "%CosmosDBDatabaseName%", collectionName = "ItemCollectionOut", connectionStringSetting = "AzureWebJobsCosmosDBConnectionString") OutputBinding<Document> outPutItem,
            final ExecutionContext context)
```

generates
```
{
  "scriptFile" : "..\\azure-functions-java-endtoendtests-1.0-SNAPSHOT.jar",
  "entryPoint" : "com.microsoft.azure.functions.endtoendtests.CosmosDBTriggerTests.CosmosTriggerAndOutput",
  "bindings" : [ {
    "type" : "cosmosDBTrigger",
    "name" : "itemIn",
    "direction" : "in",
    "databaseName" : "%CosmosDBDatabaseName%",
    "collectionName" : "ItemCollectionIn",
    "leaseCollectionName" : "leases",
    "connectionStringSetting" : "AzureWebJobsCosmosDBConnectionString",
    "createLeaseCollectionIfNotExists" : true
  }, {
    "type" : "cosmosDB",
    "name" : "itemOut",
    "direction" : "in",
    "databaseName" : "%CosmosDBDatabaseName%",
    "collectionName" : "ItemCollectionOut",
    "connectionStringSetting" : "AzureWebJobsCosmosDBConnectionString"
  } ]
}
```

Note
   **"direction" : "in",** for      **"name" : "itemOut",**

**Comment by pragnagopa** [comment]
@Flanker32 / @jdneo  - Please look into this. This is a blocker

**Comment by pragnagopa** [comment]
Please add unit test for this.

**Comment by Flanker32** [comment]
@pragnagopa  FIx in #484
